src/tests/unit/tests/background/stores/assessment-store.test.ts

```diff
@@ -93,6 +95,11 @@ describe('AssessmentStoreTest', () => {
             } as IndexedDBAPI,
             MockBehavior.Strict,
         );
+        initialAssessmentStoreDataGeneratorMock = Mock.ofType(InitialAssessmentStoreDataGenerator);
+    });
+
+    afterEach(() => {
+        initialAssessmentStoreDataGeneratorMock.verifyAll();
```

Comment:
I'm not sure I agree we should be using 'afterEach' to run the "assert" step of our tests! I know it seems easier to put this step in here but my take is this functions `afterEach`/`afterAll` should be used for clean up @dbjorge @peterdur @devaradhanm any thoughts on this?

Reply:
I agree; I would much rather make individual tests be explicit about exactly what they're arranging+acting+asserting, as a rule

Reply:
I don't know but I think the verification for the mocks is one thing that I am okay with putting in the `afterEach`.  Just saves me a sure shot repetition for something that I want to do in each test anyway!

Reply:
Putting this item as a discussion point for next engineering discussion! Will link this PR as well.